src/compas/files/__init__.py

```diff
@@ -97,19 +97,26 @@
 
 
 """
-
 from __future__ import absolute_import
-from __future__ import division
-from __future__ import print_function
-
-from .dxf import *  # noqa: F401 F403
-from .gltf import *  # noqa: F401 F403
-from .las import *  # noqa: F401 F403
-from .obj import *  # noqa: F401 F403
-from .off import *  # noqa: F401 F403
-from .ply import *  # noqa: F401 F403
-from .stl import *  # noqa: F401 F403
-from .urdf import *  # noqa: F401 F403
-from .xml import *  # noqa: F401 F403
-
-__all__ = [name for name in dir() if not name.startswith('_')]
+
+from .dxf import DXF, DXFReader, DXFParser
+from .gltf import GLTF, GLTFContent, GLTFExporter, GLTFMesh, GLTFParser, GLTFReader
```

Comment:
don't we have `force_single_line` for imports? What I did for RRC was to do `isort: skip_file` for the `__init__` because I like the grid import style (which you used on `__init__` as well), but then everything else is single-line imports